Dockerfile

```diff
@@ -22,15 +44,15 @@ ENV DEPS \
     bash \
     redis-tools
 ENV INSTALL_DEPS \
-    git
+    curl
 RUN apt-get update --quiet --quiet
 RUN apt-get upgrade --quiet --quiet
```

Comment:
We do this on lines 25/26 already. Alternatively remove the `RUN` command on line 25/26 and combine the three `RUN` commands here into one.

```suggestion
```